.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -7,8 +7,11 @@ contact_links:
     url: https://join.slack.com/t/mermaid-talk/shared_invite/enQtNzc4NDIyNzk4OTAyLWVhYjQxOTI2OTg4YmE1ZmJkY2Y4MTU3ODliYmIwOTY3NDJlYjA0YjIyZTdkMDMyZTUwOGI0NjEzYmEwODcwOTE
     about: Join our Community on Slack for Help and a casual chat.
   - name: Documentation
-    url: https://mermaid-js.github.io
+    url: https://mermaid.js.org
     about: Read our documentation for all that Mermaid.js can offer.
   - name: Live Editor
     url: https://mermaid.live
     about: Try the live editor to preview graphs in no time.
+  - name: Live Editor - Beta
+    url: https://beta.mermaid.live
+    about: Try unreleased changes in the develop branch.
```

Comment:
Just to clarify, is the `https://beta.mermaid.live` the latest `beta`/`rc` pre-released versions pushed to NPM (viewable at https://www.npmjs.com/package/mermaid?activeTab=versions)?

If so, please change this documentation to say the `Try the latest beta changes before an official release` (does this mean that `-alpha` NPM builds won't show up in `beta.mermaid.live`)

---

If the `https://beta.mermaid.live` app is actually the latest commit on the `develop` branch, I'd recommend renaming it to something like `git-develop.mermaid.live`, just to make it clear what it is!

If possible, it would be super cool if `mermaid.live`'s version number had a git hash in it too, e.g. something like:

![image](https://github.com/mermaid-js/mermaid/assets/19716675/63c7c233-b6e7-4a60-a61b-18700ea7fb7b)

Just to make it obvious which commit the `<insert-name-here>.mermaid.live` site was built-on, and to make it easier to debug if the site is still updating.

Reply:
The commit hash in version is something I had 'half-implemented' a while back in LE. 
Currently, the version numbers would mostly be wrong, as we don't update the number in the develop branch.

Reply:
> Currently, the version numbers would mostly be wrong, as we don't update the number in the develop branch.

:+1: It would be nice to have the commit hash, or maybe a warning saying that this is a development version, but that can be future work!

If you change the URL to `develop.git.mermaid.live` (or `git-develop.mermaid.live` if it's too many subdomains for your `*.mermaid.live` SSL certificate) and change the name to `Live Editor - Git `develop` preview` (or a similar name), I'd be happy for this PR to be merged.

Reply:
![Uploading image.png…]()